src/Promitor.Tests.Unit/Serialization/v1/Core/MetricDefinitionDeserializerTests.cs

```diff
@@ -26,11 +27,13 @@ public class MetricDefinitionDeserializerTests : UnitTest
         public MetricDefinitionDeserializerTests()
         {
             _azureMetricConfigurationDeserializer = new Mock<IDeserializer<AzureMetricConfigurationV1>>();
+            var logAnalyticsConfigurationDeserializer = new Mock<IDeserializer<LogAnalyticsConfigurationV1>>();
```

Comment:
Let's be consistent here:
```suggestion
            _azureMetricConfigurationDeserializer = new Mock<IDeserializer<AzureMetricConfigurationV1>>();
            _logAnalyticsConfigurationDeserializer = new Mock<IDeserializer<LogAnalyticsConfigurationV1>>();
```

Reply:
If I declared this outside and initialize it inside the constructor like your suggestion then I got warning `It can be declared as local variable` (CI will failed) so I work around with another way